suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -659,19 +701,21 @@ export const createDiscoveryThunk = createThunk(
             selectors: { selectEnabledNetworks },
         } = extra;
         const enabledNetworks = selectEnabledNetworks(getState());
-        const networks = filterUnavailableNetworks(enabledNetworks, device).map(n => n.symbol);
+        const networksSymbols = filterUnavailableNetworksCompatible(enabledNetworks, device).map(
+            n => n.symbol,
+        );
 
         dispatch(
             createDiscovery({
                 deviceState,
                 authConfirm: !device.useEmptyPassphrase,
                 index: 0,
                 status: DiscoveryStatus.IDLE,
-                total: LIMIT * networks.length,
+                total: LIMIT * networksSymbols.length,
```

Comment:
I agree, this is a mistake, thanks for catching it!
I will do it similarly as in `suite`, where I didn't forget that.. ([draft PR](https://github.com/trezor/trezor-suite/pull/14398/files/d8e95808ac13344515846ca1d08255ad06b957a4..ced3f2a8a54c2636010c61e1f11be1bce03c692c#r1764554786))

Reply:
Oh this is actually only called in `suite` desktop. To make sure, I console logged the function call, it's not called at all in `suite-native`

Reply:
I know. If it does not break desktop I am okay with that, but I have not tested that one.

Reply:
It does not, because I'm using `filterUnavailableNetworksCompatible`, which returns duplicated symbols. This is refactored in subsequent PR #14398 .

I realized that when writing the code originally, then forgot about it when I answered your comment :man_facepalming: Yes, this was working from the start, sry about confusion :heavy_check_mark: